tests/features/solution/add_solution.feature

```diff
@@ -137,6 +137,7 @@ Feature: "Add solution" visibility options.
     And I press "Add owner"
     And I press "Propose"
     Then I should see the heading "V60 filter coffee solution"
+    And I should see "Thank you for proposing a solution. Your request is currently pending approval by the site administrator."
```

Comment:
This is fine but this is asserted on the second time the solution is proposed. After it was published.
Can we assert it also during initial proposal?

Reply:
done!